pwa-devdocs/src/peregrine/testing/storybook/index.md

```diff
@@ -136,5 +135,5 @@ This command starts the Storybook application that is accessed through the brows
 http://localhost:9001/
 ```
 
-[Storybook]: https://storybook.js.org/
-[official Storybook documentation]: https://storybook.js.org/basics/introduction/
+[storybook]: https://storybook.js.org/
```

Comment:
Does it matter that the casing here doesn't match the link on line 5?

Reply:
it does not matter in markdown, but one of my linting rules is that all link references be in all lowercase.